utils/completion.py

```
@@ -1234,3 +1240,33 @@ def chat_completion_aws_bedrock_deepseek(messages, api_dict=None, aws_region="us
+    for _ in range(API_MAX_RETRY):
+        try:
+            completion = CLIENT.chat(
+                model=model,
+                messages=messages,
+                temperature=temperature,
+                max_tokens=max_tokens
+                )
+            output = {"answer": completion['choices'][0]['message']['content']}
+            break
+        except KeyError:
+            print(type(e), e)
```

Ruff (0.13.3)

Undefined name `e`

(F821)